check_code

```diff
@@ -52,6 +52,7 @@ NUMBER_OF_LINES_TO_EXAMINE = 10
 LAYERS = [
     'sample',
     'client.webApiHttpClient',
+    'client.okHttpClient'
```

Comment:
i think this is missing a comma.
and i think that's what's causing the build to fail with new messages.